src/pages/tickets.astro

```diff
@@ -0,0 +1,189 @@
+---
+import Layout from "../layouts/Layout.astro";
+import TicketTiers from "../components/ticket-tiers/ticket-tiers.astro";
+import ButtonLink from "../components/button-link/button-link.astro";
+import { Title } from "../components/typography/title";
+import BenefitsList from "../components/benefits-list/benefits-list.astro";
+import BenefitItem from "../components/benefits-list/benefit-item.astro";
+import TitleWithCta from "../components/hero-section/title-with-cta.astro";
+import { Note } from "../components/note";
+---
+
+<Layout title="Tickets | EuroPython 2025" description="tickets page">
+  <div class="container mx-auto px-4 py-8">
+    <TitleWithCta
+      title="EuroPython 2025 Tickets"
+      subtitle="Join us in Prague for Europe's premier Python conference"
+      buttonText="Register Now"
+      buttonUrl="https://tickets.europython.eu/"
+    />
+
+    <div class="my-16 text-center">
+      <Title level={2}>The What, Where and When of EuroPython 2025</Title>
+      <p class="text-xl mt-4 mb-8 max-w-3xl mx-auto">
+        EuroPython 2025 will be held on 14-20 July at the
+        <a href="/where" class="text-primary hover:text-primary-hover underline">Prague Congress Centre (PCC)</a>, Czech Republic.
+        For those who cannot physically join us but still want to support the community,
+        we have remote ticket options.
+      </p>
+
+      <div class="bg-white text-black p-6 rounded-lg shadow-md max-w-3xl mx-auto">
+        <p class="font-bold text-xl mb-4">The conference will follow this timeline:</p>
+        <ul class="text-left list-none space-y-2">
+          <li class="flex items-start">
+            <span class="flex-shrink-0 w-6">📅</span>
+            <span><strong>Two Workshop/Tutorial Days</strong> (14-15 July, Mon-Tue)</span>
+          </li>
+          <li class="flex items-start">
+            <span class="flex-shrink-0 w-6">📅</span>
+            <span><strong>Three Conference Days</strong> (16-18 July, Wed-Fri)</span>
+          </li>
+          <li class="flex items-start">
+            <span class="flex-shrink-0 w-6">📅</span>
+            <span><strong>Sprint Weekend</strong> (19-20 July, Sat-Sun)</span>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="my-16">
+      <Title level={2} className="text-center mb-8">Which Ticket Tier Should I Buy?</Title>
+
+      <div class="max-w-4xl mx-auto">
+        <p class="text-lg mb-8">
+          Each ticket type is available in three payment tiers. We encourage and trust you to pick a
+          fair ticket tier that fits your personal circumstance. The money you spend will be put
+          straight back into the community to support our diversity initiatives,
+          <a href="/finaid" class="text-primary hover:text-primary-hover underline">financial aid programmes</a> and
+          <a href="https://www.europython-society.org/grants/" class="text-primary hover:text-primary-hover underline">other grants</a>
```

Comment:
I feel here in terms of branding is nice to mention the full title of the programme
```suggestion
          <a href="https://www.europython-society.org/grants/" class="text-primary hover:text-primary-hover underline">the EPS Grants programme</a>
```